### PR TITLE
merge createRMh5parm.py with current prefactor version

### PR DESCRIPTION
**Pull request by adrabent**

### CODE REVIEW

**Review by maaijke — CHANGES_REQUESTED** [review]
Could you include again, the all_stations option in main? If you prefer you can make the default True (e.g. in the arguments change -a option to -s --single (for single station)). I can imagine situations (AARTFAAC?) where you really dont want to calculate RM per stations/antenna. Also some documentation about the proxyserver arguments of main would be nice

**Comment by adrabent** [comment]
I added the option back.

File: RMextract/LOFAR_TOOLS/createRMh5parm.py [hunk 1]
```diff
@@ -73,86 +80,179 @@ def main(MSfiles, h5parmdb, solset_name = "sol000",all_stations=False,timestep=3
...
                         help='Prefix of the IONEX files. (default: \"CODG\")')
-    parser.add_argument('--ionexpath', '--path', type=str, default='IONEXdata/',
-                        help='Directory where to store the IONEX files. (default: \"IONEXdata/\")')
+    parser.add_argument('--ionexpath', '--path', type=str, default='./',
+                        help='Directory where to store the IONEX files. (default: \"./\")')
     parser.add_argument('--solsetName', '--solset', type=str, default='sol000',
                         help='Name of the h5parm solution set (default: sol000)')
     parser.add_argument('--all','-a', help=
                         'calculate RM per station (default calculates only for CS002LBA)',
                         action='store_true',dest="allStations")
```
Comment (maaijke) [inline RMextract/LOFAR_TOOLS/createRMh5parm.py:227]:
```suggestion
                        action='store_false',dest="allStations")
```

File: RMextract/LOFAR_TOOLS/createRMh5parm.py [hunk 2]
```diff
@@ -73,86 +80,179 @@ def main(MSfiles, h5parmdb, solset_name = "sol000",all_stations=False,timestep=3
...
     parser.add_argument('--prefix', type=str, default='CODG',
                         help='Prefix of the IONEX files. (default: \"CODG\")')
-    parser.add_argument('--ionexpath', '--path', type=str, default='IONEXdata/',
-                        help='Directory where to store the IONEX files. (default: \"IONEXdata/\")')
+    parser.add_argument('--ionexpath', '--path', type=str, default='./',
+                        help='Directory where to store the IONEX files. (default: \"./\")')
     parser.add_argument('--solsetName', '--solset', type=str, default='sol000',
                         help='Name of the h5parm solution set (default: sol000)')
     parser.add_argument('--all','-a', help=
                         'calculate RM per station (default calculates only for CS002LBA)',
```
Comment (maaijke) [inline RMextract/LOFAR_TOOLS/createRMh5parm.py:226]:
```suggestion
                        'calculate RM only for CS002LBA (default calculate for all stations in the MS)',
```

File: RMextract/LOFAR_TOOLS/createRMh5parm.py [hunk 3]
```diff
@@ -73,86 +80,179 @@ def main(MSfiles, h5parmdb, solset_name = "sol000",all_stations=False,timestep=3
...
                         help='URL of the server to use. (default: ftp://ftp.aiub.unibe.ch/CODE/)')
     parser.add_argument('--prefix', type=str, default='CODG',
                         help='Prefix of the IONEX files. (default: \"CODG\")')
-    parser.add_argument('--ionexpath', '--path', type=str, default='IONEXdata/',
-                        help='Directory where to store the IONEX files. (default: \"IONEXdata/\")')
+    parser.add_argument('--ionexpath', '--path', type=str, default='./',
+                        help='Directory where to store the IONEX files. (default: \"./\")')
     parser.add_argument('--solsetName', '--solset', type=str, default='sol000',
                         help='Name of the h5parm solution set (default: sol000)')
     parser.add_argument('--all','-a', help=
```
Comment (maaijke) [inline RMextract/LOFAR_TOOLS/createRMh5parm.py:225]:
```suggestion
    parser.add_argument('--single','-s', help=
```

**Review by maaijke — CHANGES_REQUESTED** [review]
I tried to suggest changes inline to line 225-227. Not sure if this worked out as expected

**Pull request merged**